charts/castai-cluster-controller/Chart.yaml

```diff
@@ -2,5 +2,5 @@ apiVersion: v2
 name: castai-cluster-controller
 description: CAST AI cluster controller deployment chart.
 type: application
-version: 0.49.1
-appVersion: "v0.34.2"
+version: 0.49.2
```

Comment:
Nothing has changed in the chart, why version change?

Reply:
Because we are using the 2020 flow of updating helm charts :D and we need to do it manually :D

Reply:
Isn't defauolt always increase this version? I always was asked to do that

Reply:
It's not enough just to bump appVersion?

Reply:
Guess not as chart version is where release points.